pkg/cluster/nodeutils/util.go

```diff
@@ -76,13 +78,37 @@ func CopyNodeToNode(a, b nodes.Node, file string) error {
 
 // LoadImageArchive loads image onto the node, where image is a Reader over an image archive
 func LoadImageArchive(n nodes.Node, image io.Reader) error {
-	cmd := n.Command("ctr", "--namespace=k8s.io", "images", "import", "-").SetStdin(image)
+	snapshotter, err := getSnapshotter(n)
+	if err != nil {
+		return err
+	}
+	cmd := n.Command("ctr", "--namespace=k8s.io", "images", "import", "--snapshotter", snapshotter, "-").SetStdin(image)
 	if err := cmd.Run(); err != nil {
 		return errors.Wrap(err, "failed to load image")
 	}
 	return nil
 }
 
+func getSnapshotter(n nodes.Node) (string, error) {
+	out, err := exec.Output(n.Command("containerd", "config", "dump"))
+	if err != nil {
+		return "", errors.Wrap(err, "failed to detect containerd snapshotter")
+	}
+	return parseSnapshotter(string(out))
+}
+
+func parseSnapshotter(config string) (string, error) {
+	parsed, err := toml.Load(config)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to detect containerd snapshotter")
+	}
+	snapshotter, ok := parsed.GetPath([]string{"plugins", "io.containerd.grpc.v1.cri", "containerd", "snapshotter"}).(string)
```

Comment:
worth noting: `io.containerd.grpc.v1.cri` is a key in containerd config, *not* a dot delimited path, this doesn't play well with `Get` which just splits on `.`, so we need to use `GetPath` and write out each key (which, is fine)

we'll need to be careful of this if we ever switch libraries, but we have a unit test here with an actual known config.

go-toml has a v2 now, which is faster, but only does marshal/unmarshal, at some point we should figure out what maintained toml library to standardize on and rework the toml patching code we have as well ... but that's not for today 🙃